src/net/textproto/reader.go

```diff
@@ -481,6 +481,15 @@ func (r *Reader) ReadDotLines() ([]string, error) {
 //	}
 //
 func (r *Reader) ReadMIMEHeader() (MIMEHeader, error) {
+	header, _, err := r.readMIMEHeader(false /* cfRecordRequestLines */)
+	return header, err
+}
+
+func (r *Reader) CFReadMIMEHeader() (MIMEHeader, []CFHeaderLine, error) {
+	return r.readMIMEHeader(true /* cfRecordRequestLInes */)
```

Comment:
`I` in `LInes is accidentally capitalized. However I believe that you have to expose this bool parameter anyway (and set it to the http.Server parameter).

Reply:
Added `recordHeaderLInes bool` parameter to `CFReadMIMEHeader`. In net/http this is set to `Server.CFRecordRecordLInes`.